<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="windowTitle">
   <string>Porypalette</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="control_layout">
     <item>
      <widget class="QGroupBox" name="processing_group">
       <property name="title">
        <string>Image Processing</string>
       </property>
       <layout class="QHBoxLayout" name="processing_layout">
        <item>
         <widget class="QPushButton" name="btn_tileset">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumHeight">
           <number>40</number>
          </property>
          <property name="text">
           <string>‚öôÔ∏è Load Overworld Tileset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_target">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumHeight">
           <number>40</number>
          </property>
          <property name="text">
           <string>üé® Load Target Sprite</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_save">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumHeight">
           <number>40</number>
          </property>
          <property name="text">
           <string>üíæ Save Selected</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_toggle_theme">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumHeight">
           <number>40</number>
          </property>
          <property name="text">
           <string>üí°</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="content_layout">
     <item stretch="30">
      <layout class="QVBoxLayout" name="original_box">
       <item>
        <widget class="QLabel" name="label_original">
         <property name="text">
          <string>Original Sprite</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="original_view">
         <property name="renderHints">
          <set>QPainter::Antialiasing</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item stretch="70">
      <layout class="QGridLayout" name="converted_box">
       <property name="horizontalSpacing">
        <number>15</number>
       </property>
       <item row="0" column="0" colspan="4">
        <widget class="QLabel" name="label_instruction">
         <property name="text">
          <string>Click to select conversion (Green = Selected, Blue = Most Diverse)</string>
         </property>
        </widget>
       </item>
       <!-- Note: The dynamic result views will need to be added programmatically -->
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>